src/style.css

```diff
@@ -0,0 +1,125 @@
+@font-face {
+  font-family: Avenir-book;
+  src: url(./fonts/Avenir-Book.ttf) format("truetype");
+  font-weight: lighter;
+}
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf) format("truetype");
+  font-weight: bold;
+}
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc) format("truetype");
+  font-weight: normal;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
```

Comment:
you haven't included font "Avenir" with weight 300 that's why it's not necessary.